Modules/_ctypes/callproc.c

```diff
@@ -648,14 +648,14 @@ union result {
     int i;
     long l;
     long long q;
-    long double D;
+    long double g;
```

Comment:
@vstinner, this is something I worry about here.

IIUIC, this is a private interface and we can rename it's members.  It seems this is not used across the codebase, but I appreciate a second look.  (No tests seems to be affected, though maybe it's just lack of test coverage.)

Reply:
> I would suggest to remove it

I'm not sure.  It seems, naming of union members aren't used - only size is important.

As long as we keep one-letter codes, I think it's good to have corresponding members in this struct.

Same holds for `struct tagPyCArgObject` - I forgot to update that.  I'll make a patch.  BTW do we need different (wrt `union result`) unnamed union here?  I doubt.